omod/src/main/webapp/resources/htmlforms/section-history.xml

```diff
@@ -192,25 +195,25 @@
         <section id="complaint" sectionTag="fieldset" headerTag="legend" headerStyle="title"
                  headerCode="pihcore.refer.title">
             <div class="section-container">
-                <p>
-                    <label>
-                        <uimessage code="pihcore.refer.date"/>
-                    </label>
-                    <obs conceptId="CIEL:163181" allowTime="false" />
-                </p>
-
                 <p>
                     <obs conceptId="PIH:Type of referring service"
                          answerConceptId="PIH:SELF-REFER"
                          answerCode="pihcore.selfRefer"
                          style="checkbox" />
                 </p>
 
-            <div class="two-columns">
                 <p>
                     <label>
-                        <uimessage code="pihcore.refer.referBy"/>
+                        <uimessage code="pihcore.refer.date"/>
                     </label>
+                    <obs conceptId="CIEL:163181" allowTime="false" />
+                </p>
+
+            <div class="two-columns">
```

Comment:
We seem to have some weird indentation going on... would you mind doing an auto-reformat prior to merging?